examples/algorithms/plot_dedensification.py

```diff
@@ -0,0 +1,104 @@
+"""
+=============
+Dedensification
+=============
+
+Examples of dedensification of a graph.  Dedensification retain the structural
+pattern of the original graph will only add compressor nodes when doing so
+would result in fewer edges in the compressed graph. Both methods retain the
+structural pattern of the original graph.
```

Comment:
Grammar check this paragraph.

Reply:
You're right, this needed to be changed.  It's now been updated to:

> Examples of dedensification of a graph.  Dedensification retains the structural pattern of the original graph and will only add compressor nodes when doing so would result in fewer edges in the compressed graph.